ArduPlane/quadplane.h

```diff
@@ -410,6 +410,7 @@ class QuadPlane
     enum tailsitter_input {
         TAILSITTER_INPUT_MULTICOPTER = 0,
         TAILSITTER_INPUT_PLANE       = 1,
+        TAILSITTER_INPUT_BF_ROLL     = 2,
```

Comment:
you need to also add this to the param description

Reply:
Thanks, added.
Please let me know if  my description makes sense. I altered the description for FixedWingInput a little too. Also, should the description values be MultiCopter, Plane, BF_ROLL to align with the enum values? I chose to maintain the existing convention since this is user-facing.

Reply:
maybe the description should be 'MultiCopter with Body frame roll' although I don't think it matters too much. Apart from that it looks good.